tests/conference/test_conference_service.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+
+def test_conference_create(session):
+    from dispatch.conference.service import create
+    from dispatch.conference.models import ConferenceCreate
+
+    resource_id = "000000"
+    resource_type = "resourcetype"
+    weblink = "https://www.example.com"
+    conference_id = "12345"
+    conference_challenge = "a0v0a0v9a"
+
+    conference_in = ConferenceCreate(
+        resource_id=resource_id,
+        resource_type=resource_type,
+        weblink=weblink,
+        conference_id=conference_id,
+        conference_challenge=conference_challenge,
+    )
+    conference = create(db_session=session, conference_in=conference_in)
+    assert conference
+    assert conference.resource_id == "000000"
+    assert resource_type == "resourcetype"
```

Comment:
Did you forget to prefix `resource_type` with `conference.`?

Reply:
Yes thanks, that would always succeed, so incorrect. Fixed in 6703f56.